guides/release/components/actions-and-events.md

```diff
@@ -297,7 +297,7 @@ import { action } from '@ember/object';
 
 export default class SendMessage extends Component {
   @action
-  sendMessage(messageType) {
+  async sendMessage(messageType) {
```

Comment:
I know some work supporting `async`/`await` landed into backburner and Ember in 3.x, but I'm not sure if FW core is encouraging this in app code yet. I've queried a few people for clarification.

Regardless, I'm not sure why this action is `async`... Simply to provide an example that it is possible?

Reply:
I've been using async/await _everywhere_ in emberclear, and haven't ran into issues. :man_shrugging: 

I think the the name implies that there is going to be some interaction with a backend, so I would expect it to be async :)